setup.py

```diff
@@ -28,6 +28,7 @@
     "all_backends": ["cupy", "torch"],
     "viz": ["matplotlib"],
     "test": ["pytest", "cupy", "torch"],
+    "travis": ["pytest", "matplotlib"],
```

Comment:
Could we try to add `torch`, if we fix the hanging in `torch_cuda` ?

Reply:
yes!